packages/definitions-parser/src/get-affected-packages.ts

```diff
@@ -52,6 +56,11 @@ export async function getAffectedPackages(
       )
     );
   }
+
+  console.log("changedPackageDirectories", changedPackageDirectories);
+  console.log("addedPackageDirectories", addedPackageDirectories);
+  console.log("allDependentDirectories", allDependentDirectories);
```

Comment:
Don't we log similar derived information a little bit later? It's useful but probably overwhelming especially for large changes.

No recommendation, just an observation. More logging is better than none, so I guess leave it in.

Reply:
forgot to push, this is not supposed to be there!

Reply:
(pushed now...)